framework/SupervisedLearning/ARMA.py

```diff
@@ -278,7 +278,7 @@ def __init__(self):
     self.zeroFilterTarget  = None # target for whom zeros should be filtered out
     self.zeroFilterTol     = None # tolerance for zerofiltering to be considered zero, set below
     self._masks            = collections.defaultdict(dict)   # dictionay of masks, including zeroFilterMask(where zero), notZeroFilterMask(Where non zero), and maskPeakRes.
-    self._minBins          = 20   # min number of bins to use in determining distributions, eventually can be user option, for now developer's pick
+    self._minBins          = 50   # min number of bins to use in determining distributions, eventually can be user option, for now developer's pick
```

Comment:
I think you'll want to revert this change, otherwise we'll have to regold all the ARMA regression tests.